zebra-rpc/src/methods.rs

```diff
@@ -103,6 +105,22 @@ pub trait Rpc {
     /// zcashd reference: [`getrawmempool`](https://zcash.github.io/rpc/getrawmempool.html)
     #[rpc(name = "getrawmempool")]
     fn get_raw_mempool(&self) -> BoxFuture<Result<Vec<String>>>;
+
+    /// Returns the raw transaction data, as a [`GetRawTransaction`] JSON string or structure.
+    ///
+    /// zcashd reference: [`getrawtransaction`](https://zcash.github.io/rpc/getrawtransaction.html)
+    ///
+    /// # Parameters
+    ///
+    /// - `txid`: (string, required) The transaction ID of the transaction to be returned.
+    /// - `verbose`: (numeric, optional, default=0) If 0, return a string of hex-encoded data, otherwise return a JSON object.
+    /// - `blockhash`: (string, optional) The block in which to look for the transaction.
```

Comment:
```suggestion
```
This argument is not used so we should not document it. 

Also please add a `# Notes` section explaining what parameters we are supporting and what result fields we expose. Check other RPC methods for it, for example: https://github.com/ZcashFoundation/zebra/blob/main/zebra-rpc/src/methods.rs#L83-L91